warp10/src/main/java/io/warp10/continuum/egress/EgressFetchHandler.java

```diff
@@ -1365,38 +1367,42 @@ static void jsonDump(PrintWriter pw, Iterator<GTSDecoder> iter, long now, long c
               sb.append(",");
             }
             
-            //sb.append(gson.toJson(entry.getKey()));
-            sb.append(serializer.serialize(entry.getKey()));
+            sb.append(JsonUtils.objectToJson(entry.getKey()));
             sb.append(":");
-            //sb.append(gson.toJson(entry.getValue()));
-            sb.append(serializer.serialize(entry.getValue()));
+            sb.append(JsonUtils.objectToJson(entry.getValue()));
             first = false;
           }
           sb.append("}");
-          
-          sb.append(",\"a\":{");
+
+          sb.append(",\"");
+          sb.append(MetadataSerializer.FIELD_ATTRIBUTES);
+          sb.append("\":{");
 
           first = true;
           for (Entry<String, String> entry: decoder.getMetadata().getAttributes().entrySet()) {
             if (!first) {
               sb.append(",");
             }
             
-            //sb.append(gson.toJson(entry.getKey()));
-            sb.append(serializer.serialize(entry.getKey()));
+            sb.append(JsonUtils.objectToJson(entry.getKey()));
             sb.append(":");
-            //sb.append(gson.toJson(entry.getValue()));
-            sb.append(serializer.serialize(entry.getValue()));
+            sb.append(JsonUtils.objectToJson(entry.getValue()));
             first = false;
           }
           
           sb.append("}");
-          sb.append(",\"i\":\"");
+          sb.append(",\"");
+          sb.append(MetadataSerializer.FIELD_LABELSID);
```

Comment:
It is a an id, not directly the labelsId, so better call this field FIELD_ID as it could very well not stay related to the labels id only

Reply:
True. As this is not related to Metadata, I have to put this constant in EgressFetchHandler.